src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message in the `doWork` method should be more informative and indicate that it's a Truck that is working. For example, it could say 'Truck is doing its work.'